### ISSUE TITLE
Can't connect to linux nodes from windows

### ISSUE THREAD

**Issue by acorbellini** [JuliaLang/julia#2902]
Hi!
 I'm currently trying Julia under Windows and wanted to point out that the machinefile option is not working correctly (I guess). I used the binary distribution and the built from the source, but when I try to connect to an ubuntu node, it tries to find Julia under the Windows installation path (but on ubuntu). I've set JULIA_HOME on the ubuntu box, but it doesn't seem to be using it.
I'm able to use this feature on Ubuntu, but it would be nice to be able to spawn jobs from windows to ubuntu.
Thanks!!

**Comment by mgkuhn** [comment]
I have successfully used Julia 1.6-DEV (following #38577) to invoke workers between Linux (POSIX shell) and Windows (cmd.exe shell) in all four possible directions.

Yes, you always need to specify `dir` when crossing a Windows/Linux boundary, because pathnames will be different between both worlds (e.g., drive letter) and the default `pwd()` is very unlikely to work.

`JULIA_HOME` no longer exists and Julia can usually figure out on its own where it was installed.

Julia 1.6-DEV (since #38353) now also supports passing environment variables via ssh, using the new `env` parameter to `addprocs`, which simply injects the shell commands needed to set environment variables before starting a Julia worker.

I don't know about the state of the 32/64-bit data serialization issue mentioned by @StefanKarpinski. This would warrant opening a separate issue if is is still acute.

Otherwise I think the original issue is resolved.
